### PR TITLE
Feature/wizard hosted creation error focus

### PR DESCRIPTION
**Pull request by marie-j**
### Description of the Change

Before: The error message would display at the top of the page no matter what step the scroll focused on 
Now: The error message displays before the step on which the focus is

### CODE REVIEW

File: src/exchange/wizard-hosted-creation/wizard-message/wizard-message.html [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+       </button>
+
+       <p data-ng-bind-html="$ctrl.homepage.messaging.message"></p>
+
+       <ul class="mb-1 ml-2" data-ng-if="$ctrl.homepage.messaging.messageDetails != null && $ctrl.homepage.messaging.messageDetails.length > 0">
+            <li data-ng-repeat="message in $ctrl.homepage.messaging.messageDetails track by $index">
+                <strong data-ng-if="message.id != null">{:: {message.id}}:</strong> {{:: message.message}}
+            </li>
+        </ul>
+    </div>
```
Comment (antleblanc) [inline src/exchange/wizard-hosted-creation/wizard-message/wizard-message.html]: missing `insert_final_newline`

File: src/exchange/wizard-hosted-creation/wizard-message/wizard-message.html [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+            type="button"
+            aria-label="tr('wizard_close')"
+            data-ng-click="$ctrl.homepage.closeMessages()">
+       </button>
+
+       <p data-ng-bind-html="$ctrl.homepage.messaging.message"></p>
+
+       <ul class="mb-1 ml-2" data-ng-if="$ctrl.homepage.messaging.messageDetails != null && $ctrl.homepage.messaging.messageDetails.length > 0">
+            <li data-ng-repeat="message in $ctrl.homepage.messaging.messageDetails track by $index">
+                <strong data-ng-if="message.id != null">{:: {message.id}}:</strong> {{:: message.message}}
```
Comment (antleblanc) [inline src/exchange/wizard-hosted-creation/wizard-message/wizard-message.html]: typo?

`{:: {` => `{{::`

File: src/exchange/wizard-hosted-creation/wizard-message/wizard-message.html [hunk 3]
```diff
@@ -0,0 +1,17 @@
+<div class="alert alert-dismissible"
+    data-ng-class="$ctrl.homepage.messaging.alertType"
+    id="wizard-error-message">
+        <button class="close"
```
Comment (antleblanc) [inline src/exchange/wizard-hosted-creation/wizard-message/wizard-message.html]: indentation issue?

File: src/exchange/wizard-hosted-creation/wizard-message/wizard-message.html [hunk 4]
```diff
@@ -0,0 +1,17 @@
+<div class="alert alert-dismissible"
```
Comment (antleblanc) [inline src/exchange/wizard-hosted-creation/wizard-message/wizard-message.html]: +`role="alert"`

**Comment by jleveugle** [comment]
@marie-j some news from QC?

**Pull request merged**